notebook/static/tree/js/main.js

```diff
@@ -37,6 +37,7 @@ require([
     loginwidget){
     "use strict";
 
+    //gets notebooklist creates a method
```

Comment:
Thanks for the feedback. Any input on the jquery I wrote and how to
refactor it?
On Jul 21, 2015 2:34 PM, "Min RK" <notifications@github.com> wrote:

> In notebook/static/tree/js/main.js
> <https://github.com/jupyter/notebook/pull/217#discussion_r35157436>:
>
> > @@ -37,6 +37,7 @@ require([
> >      loginwidget){
> >      "use strict";
> >
> > +    //gets notebooklist creates a method
>
> this comment isn't accurate
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/217/files#r35157436>.
>